player/tests/data/buffer-zero-init-for-binding.ron

```diff
@@ -0,0 +1,99 @@
+(
+    features: (bits: 0x0),
+    expectations: [
+        (
+            name: "buffer has correct values",
+            buffer: (index: 0, epoch: 1),
+            offset: 0,
+            data: Raw([0x00, 0x00, 0x00, 0x00,
+                       0x01, 0x00, 0x00, 0x00,
+                       0x02, 0x00, 0x00, 0x00,
+                       0x03, 0x00, 0x00, 0x00]),
+        )
+    ],
+    actions: [
+        CreateShaderModule(

```

Comment:
If you are trying to ensure that bindings get zeroed out, you don't actually need shaders. Any bindings done inside a render pass are considered a part of the sync scope. So you can just start a dummy render pass, bind something, and end the pass, and you should be able to test the effect of these bindings (without any draws).

Reply:
Note quite true I believe. Let's say zero out on mapping works perfectly fine but our zero out on binding completely broke. Then just doing the binding will not zero the buffer out but later when we check the result we still get a zeroed out buffer. Thus only testing what the existing test already tested.

By having a shader that produces a different result depending on was in the buffer at the time of the shader invocation, we can reason about the zeroing that should have occurred because of the binding.

Reply:
That's good thinking, you are right!